packages/netlify-cms-backend-bitbucket/src/implementation.ts

```diff
@@ -495,37 +490,47 @@ export default class BitbucketBackend implements Implementation {
         branches.map(branch => contentKeyFromBranch(branch)),
       );
 
-    const readUnpublishedBranchFile = (contentKey: string) =>
-      this.api!.readUnpublishedBranchFile(contentKey);
+    const ids = await unpublishedEntries(listEntriesKeys);
+    return ids;
+  }
 
-    return unpublishedEntries(listEntriesKeys, readUnpublishedBranchFile, API_NAME);
+  async unpublishedEntry({
+    id,
+    collection,
+    slug,
+  }: {
+    id?: string;
+    collection?: string;
+    slug?: string;
+  }) {
+    if (id) {
+      const data = await this.api!.retrieveUnpublishedEntryData(id);
+      return data;
+    } else if (collection && slug) {
+      const entryId = generateContentKey(collection, slug);
+      const data = await this.api!.retrieveUnpublishedEntryData(entryId);
+      return data;
+    } else {
+      throw new Error('Missing unpublished entry id or collection and slug');
+    }
   }
 
-  async unpublishedEntry(
```

Comment:
Moved reading the content file and media file to designated methods `unpublishedEntryDataFile` and `unpublishedEntryMediaFile` to give `backend.js` more control on which file to read